src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {App} from './App';
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
```

Comment:
```suggestion
it('should render app container', () => {
```
As a rule after it 'should' word comes, and don't use 'correctly' word